src/main/webapp/resources/js/pages/analysis/components/AnalysisDetails.jsx

```diff
@@ -0,0 +1,223 @@
+import React, { useContext, Suspense, lazy, useEffect } from "react";
+import {
```

Comment:
Comments

Reply:
Fixed in f8844707a5f60a885ba87a349edb62b076eaba15